{
  "uid" : "446684ea2b183a46",
  "name" : "div_pythoncode_0",
  "fullName" : "test_Calculator.TestCalc#test_div_int_zero",
  "historyId" : "786ddc6dbd02a543a4eeb939e2586d6e",
  "time" : {
    "start" : 1612420386974,
    "stop" : 1612420386974,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "TypeError: unsupported operand type(s) for /: 'str' and 'int'",
  "statusTrace" : "self = <testing.test_Calculator.TestCalc object at 0x000001564E9C68C8>, get_instance = <pythoncode.Calculator.Calculator object at 0x000001564F491FC8>, get_div_int_error_datas_with_fixture = ['pythoncode', 0, 'N/A']\n\n    @allure.title(\"div_{get_div_int_error_datas_with_fixture[0]}_{get_div_int_error_datas_with_fixture[1]}\")\n    @allure.story(\"div Integer error Function\")\n    def test_div_int_zero(self, get_instance, get_div_int_error_datas_with_fixture):\n        div_int_zero = get_div_int_error_datas_with_fixture\n        with pytest.raises(ZeroDivisionError):  # pytest inner function\n>           get_instance.div(div_int_zero[0], div_int_zero[1])\n\ntest_Calculator.py:77: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <pythoncode.Calculator.Calculator object at 0x000001564F491FC8>, a = 'pythoncode', b = 0\n\n    def div(self, a, b):\n        logger.info(f\"This case is to test {a} / {b}\")\n>       return a / b\nE       TypeError: unsupported operand type(s) for /: 'str' and 'int'\n\n..\\pythoncode\\Calculator.py:21: TypeError",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "get_div_int_error_datas_with_fixture",
    "time" : {
      "start" : 1612420386974,
      "stop" : 1612420386974,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "hasContent" : false,
    "stepsCount" : 0,
    "shouldDisplayMessage" : false,
    "attachmentsCount" : 0
  }, {
    "name" : "get_instance",
    "time" : {
      "start" : 1612420386974,
      "stop" : 1612420386974,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "hasContent" : false,
    "stepsCount" : 0,
    "shouldDisplayMessage" : false,
    "attachmentsCount" : 0
  } ],
  "testStage" : {
    "status" : "broken",
    "statusMessage" : "TypeError: unsupported operand type(s) for /: 'str' and 'int'",
    "statusTrace" : "self = <testing.test_Calculator.TestCalc object at 0x000001564E9C68C8>, get_instance = <pythoncode.Calculator.Calculator object at 0x000001564F491FC8>, get_div_int_error_datas_with_fixture = ['pythoncode', 0, 'N/A']\n\n    @allure.title(\"div_{get_div_int_error_datas_with_fixture[0]}_{get_div_int_error_datas_with_fixture[1]}\")\n    @allure.story(\"div Integer error Function\")\n    def test_div_int_zero(self, get_instance, get_div_int_error_datas_with_fixture):\n        div_int_zero = get_div_int_error_datas_with_fixture\n        with pytest.raises(ZeroDivisionError):  # pytest inner function\n>           get_instance.div(div_int_zero[0], div_int_zero[1])\n\ntest_Calculator.py:77: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <pythoncode.Calculator.Calculator object at 0x000001564F491FC8>, a = 'pythoncode', b = 0\n\n    def div(self, a, b):\n        logger.info(f\"This case is to test {a} / {b}\")\n>       return a / b\nE       TypeError: unsupported operand type(s) for /: 'str' and 'int'\n\n..\\pythoncode\\Calculator.py:21: TypeError",
    "steps" : [ ],
    "attachments" : [ {
      "uid" : "47def9d7ffd9cbc7",
      "name" : "log",
      "source" : "47def9d7ffd9cbc7.txt",
      "type" : "text/plain",
      "size" : 77
    } ],
    "parameters" : [ ],
    "hasContent" : true,
    "stepsCount" : 0,
    "shouldDisplayMessage" : true,
    "attachmentsCount" : 1
  },
  "afterStages" : [ {
    "name" : "get_instance::0",
    "time" : {
      "start" : 1612420386979,
      "stop" : 1612420386979,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "hasContent" : false,
    "stepsCount" : 0,
    "shouldDisplayMessage" : false,
    "attachmentsCount" : 0
  } ],
  "labels" : [ {
    "name" : "feature",
    "value" : "计算器"
  }, {
    "name" : "story",
    "value" : "div Integer error Function"
  }, {
    "name" : "suite",
    "value" : "test_Calculator"
  }, {
    "name" : "subSuite",
    "value" : "TestCalc"
  }, {
    "name" : "host",
    "value" : "DESKTOP-ALON4FT"
  }, {
    "name" : "thread",
    "value" : "10176-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "test_Calculator"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ {
    "name" : "get_div_int_error_datas_with_fixture",
    "value" : "['pythoncode', 0, 'N/A']"
  } ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ ]
  },
  "source" : "446684ea2b183a46.json",
  "parameterValues" : [ "['pythoncode', 0, 'N/A']" ]
}